policy/test/inputs/parsec-cca-result.json

```diff
@@ -0,0 +1,24 @@
+{
+    "eat_profile": "tag:github.com,2023:veraison/ear",
+    "iat": 1666091373,
+    "ear.verifier-id": {
+      "build": "test",
+      "developer": "test"
+    },
+    "submods": {
+      "test": {
```

Comment:
[silly comment] not sure what is the purpose of this test vector, but shouldn't this be parsec-cca (or similar)?